pages/parent/enrollment.tsx

```diff
@@ -23,10 +25,14 @@ type ParentEnrollClassProps = {
 export default function ParentEnrollClass({
     session,
 }: ParentEnrollClassProps): JSX.Element {
-    const [pageNum, setPageNum] = useState<number>(0);
     const [selectedChild, setSelectedChild] = useState<number>(0);
     const { user, isLoading, error } = useUser(session.id as string);
     const router = useRouter();
+    const { classId, page } = router.query;
+    const numberClassId =
+        classId && classId.length > 0 ? Number(classId) : null;
+    const numberPage = page ? parseInt(String(page), 10) : null;
```

Comment:
oh my bad! I guess we can do this for classId as well just to be consistent:
```typescript
    const numberClassId = classId ? parseInt(classId as string, 10) : null;
```

Sorry bout the nitpick. While we are at it, maybe we can do a `router.back()` if classId is null since it doesn't make sense to continue in that case.